lib/nodeSass/UndefinedVariableFixer.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs');
+const BaseFixer = require('./BaseFixer');
+
+class UndefinedVariableFixer extends BaseFixer {
```

Comment:
Maybe some unit tests to add for this case?

Reply:
Unit tests complexity is very high over here. 
Integration tests could be way easier to implement, but there is still concerns on how to make the run idempotence (manual revert each time? or create a system to copy files).